### DISCUSSION TITLE
Why data binding only works at top level?

### DISCUSSION THREAD

**Discussion by alvinsw** [describo/describo.github.io#6] [General]
Hi Marco,

I have tried using the crate-builder-component in a new simple Vue app.
I managed to get it running and displaying the crate data.
However I just found out that the crate data is bound only at top level.
For example, if you have `data.crate = { ... }` and pass it to `:crate="data.crate"`, you can't do for example: 
`data.crate['@graph'][7].name = 'abc'`, nor `data.crate['@graph'][7].name` is updated when the value in the text box is changed.
I know that we can use the @save:crate event to get the current/modified data. 
But, is there a reason for not properly bound each nested property to its corresponding ui component?
It seems to be counterintuitive to how Vue component generally works.

**Comment by marcolarosa** [comment]
As this is relevant to the component please move the question to that repo: https://github.com/describo/crate-builder-component

**Reply by alvinsw** [reply]
There is no discussion on the repo, so I have posted it as an issue instead.

**Comment by marcolarosa** [comment]
Moved to https://github.com/describo/crate-builder-component/issues/31
